### ISSUE TITLE
[Mechanics] Thorns deals incorrect base damage and scales too little from SP

### ISSUE THREAD

**Issue by Chenocide** [dalaranwow/dalaran-wow#5726]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Level 8 Thorns deals 68 nature damage with 0 spell power and the spell power scaling seems too low and random. 
![](https://i.imgur.com/JqM31Zf.png)

**Expected behaviour**: Tell us what should happen instead

Base damage should be 73 and the spell power scaling should be consistently 3.3%
I haven't tested ranks 1-7 and I haven't tested with the druid talent Brambles, which increases Thorns damage with 25%/50%/75%. I assume that all of the ranks have issues but Brambles should be straight forward since it's just an overall damage increase that can be multiplied onto the calculated damage. 

**Steps to reproduce the problem**:

1. Equip a set of gear with spell power
2. Calculate the expected damage with the formula below
3. Watch the in-game result and compare values. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Formula explained:
![](https://i.imgur.com/ywBCQfm.png)

Wowwiki SP scaling 3.3%, old revision from 2009:
https://wowwiki.fandom.com/wiki/Thorns?oldid=2064953

Formula used for finding the approximate coeff:
(observed damage-base damage)/spell power

Example:
(96-68)/939 = 0.0298 or 2.98%

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Rank 8 Thorns:
53307

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Examples with different amount of spell power and the observed coeff. 
![](https://i.imgur.com/QZiVvpZ.png)
The coeff is all over the place and is too low in every example.
